src/pages/settings/Subscription/SubscriptionSettingsPage.tsx

```diff
@@ -18,6 +26,10 @@ function SubscriptionSettingsPage() {
                 shouldShowBackButton={isSmallScreenWidth}
                 icon={Illustrations.CreditCardsNew}
             />
+            <ScrollView>
```

Comment:
We can reuse the same styles since the upper structure is the same (as here - https://github.com/Expensify/App/blob/d7fd40a07da4437b58dba8e3b77c621e6967edea/src/pages/settings/Wallet/WalletPage/WalletPage.tsx#L363)

Reply:
Done